Sources/Error Handling/ErrorUtils.swift

```diff
@@ -286,7 +286,6 @@ enum ErrorUtils {
      * If the error is already a `PurchasesError`, this simply returns the same value,
      * otherwise it wraps it into a ``ErrorCode/unknownError``.
      */
-    @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
```

Comment:
Leftover from before?

Reply:
Yeah it was only used from another method with the same availability so I didn't notice it didn't need it.